packages/inno/oh-my-posh.iss

```diff
@@ -9,6 +9,7 @@ AppPublisherURL=https://ohmyposh.dev
 AppSupportURL=https://github.com/JanDeDobbeleer/oh-my-posh/issues
 LicenseFile="bin\COPYING.txt"
 OutputBaseFilename=install
+UninstallDisplayIcon={app}\oh-my-posh.exe
```

Comment:
```suggestion
UninstallDisplayIcon={app}\bin\oh-my-posh.exe
```
The correct path for the icon is located in the `bin\` folder, represented as `{app}\bin\oh-my-posh.exe`.

Reply:
You are right, sorry for my carelessness.